smartling-files-api/src/main/java/com/smartling/api/files/v2/resteasy/ext/TranslatedFileMultipart.java

```diff
@@ -0,0 +1,30 @@
+package com.smartling.api.files.v2.resteasy.ext;
+
+import lombok.Getter;
+import org.apache.commons.io.IOUtils;
+import org.jboss.resteasy.plugins.providers.multipart.MultipartInput;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import java.io.InputStream;
+
+public class TranslatedFileMultipart
+{
+    @Getter
+    InputStream fileBody;
+    @Getter
+    MultivaluedMap<String, String> fileHeaders;
+    @Getter
+    MediaType fileMediaType;
+    @Getter
+    TranslatedFileMetadata translatedFileMetadata;
+    MultipartInput multipartInput;
+
+    public void close()
+    {
+        if (null != fileBody)
+            IOUtils.closeQuietly(fileBody);
```

Comment:
`IOUtils.closeQuietly` processes `null` fine. No need to check.

Reply:
yeah... you're right. removed redundant null-checking.